src/interceptors/WebSocket/utils/events.ts

```diff
@@ -1,3 +1,38 @@
+const kCancelable = Symbol('kCancelable')
+const kDefaultPrevented = Symbol('kDefaultPrevented')
+
+/**
+ * A `MessageEvent` superset that supports event cancellation
+ * in Node.js. It's rather non-intrusive so it can be safely
+ * used in the browser as well.
+ *
+ * @see https://github.com/nodejs/node/issues/51767
+ */
+export class CancelableMessageEvent<T = any> extends MessageEvent<T> {
```

Comment:
This is something we need to remove in the future.

TL;DR Node's `MessageEvent` cannot be canceled. Doesn't respect the `cancelable` attribute and so `preventDefault()` does nothing. Looks like a bug in Node.js. Reported.